src/utils/validation.ts

```diff
@@ -36,6 +36,17 @@ export const validateIsFutureDate = (value: string) => {
   return isFuture(parsedDate) ? undefined : 'Card expired';
 };
 
+export const isPositiveInteger = (entity: string) => (value: string) => {
```

Comment:
Shouldn't be a string. Find API that formats value before the moment when it's written to the form state.

Reply:
Also rename me to `validateIsPositiveInteger`